examples/pipeline-with-automatic-deployment-creation.yaml

```diff
@@ -0,0 +1,69 @@
+# Defining deployments in the YAML is optional but allows Valohai platform to
+# initialize deployments automatically when a related pipeline is run.
+#
+# You may reference this deployment definition in the pipeline deployment
+# nodes like further down below.
+- deployment:
```

Comment:
I kind of want to call this `deployment-template` 🤔 because the other things here are pretty concrete and, like, _certainly_ there to choose from, but this'd only materialize into a deployment if a deployment node needs it?

Is that too verbose? 🤔